modules/gitops-release-notes-1-10-5.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-10-5_{context}"]
+= Release Notes for {gitops-title} 1.10.5
+
+{gitops-title} 1.10.5 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-10-5_{context}"]
+== Errata updates
+
+[id="rhsa-2024:2817-gitops-1-10-5-security-update-advisory_{context}"]
+=== RHSA-2024:2817 - {gitops-title} 1.10.5 security update advisory
+
+Issued: 2024-05-09
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:2817[RHSA-2024:2817]
+
+If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
+----
+
+[id="fixed-issues-1-10-5_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE and EKS clusters because this binary was not available in the {gitops-shortname} container. This update fixes the issue by adding the `argocd-k8s-auth` binary in the {gitops-shortname} container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
```

Comment:
@Dhruv-Soni11 please expand GKE and EKS here

Reply:
Fixed.